cfg/param_update.yaml

```diff
@@ -0,0 +1,11 @@
+# File types
+file_common: "_common.cpp"
+file_ros: "_ros.cpp"
+file_cfg: ".cfg"
+file_yaml: ".yaml"
+
+# Path to yaml folder
+path_yaml_folder: "/home/srd-rd/catkin_ws/src/ipa_navigation/ipa_navigation_config/config/components"
```

Comment:
Added another parameter 'path_yaml_pkg' which includes the pkg name now and the 'path_yaml_folder just includes the path to the folder within the package now.